phpmyadmin/content.md

```diff
@@ -101,10 +124,14 @@ $ docker run --name myadmin -d -e PMA_PASSWORD_FILE=/run/secrets/db_password.txt
 
 #### Variables that can be read from a file using `_FILE`
 
--	PMA_PASSWORD
--	MYSQL_ROOT_PASSWORD
--	MYSQL_PASSWORD
--	PMA_HOSTS
--	PMA_HOST
+-	`PMA_PASSWORD`
+-	`MYSQL_ROOT_PASSWORD`
+-	`MYSQL_PASSWORD`
+-	`PMA_HOSTS`
+-	`PMA_HOST`
 
 For more detailed documentation see https://docs.phpmyadmin.net/en/latest/setup.html#installing-using-docker
+
+Please report any issues with the Docker container to https://github.com/phpmyadmin/docker/issues
+
+Please report any issues with phpMyAdmin to https://github.com/phpmyadmin/phpmyadmin/issues
```

Comment:
IMO these two links would be a better fit for "Where to file issues" in the "Quick reference" which can be customized as well (although I'd add that in our experience, a large number of users can't really tell the difference between "issue with XYZ software" and "issue with the Dockerization of XYZ software" but perhaps YMMV :sweat_smile:).